README.md

```diff
@@ -36,26 +40,37 @@ module.exports = {
     charset: "utf8",
   },
 
-  plugins: ["./custom-rules"],
+  // Schemas to lint.
+  schemas: [{ name: "public" }],
 
+  // Rules to be checked. The key is the rule name and the value is an array
+  // whose first value is the severity (only "error" is supported) and the
+  // rest are rule-specific parameters.
```

Comment:
The explanation of rules, ignores and plugins seems a bit long. Perhaps it would be easier to read if the explanation here were simpler and more detailed elsewhere.